src/main/kotlin/no/nav/familie/ba/sak/kjerne/automatiskvurdering/FiltreringsreglerService.kt

```diff
@@ -0,0 +1,53 @@
+package no.nav.familie.ba.sak.kjerne.automatiskvurdering
+
+import no.nav.familie.ba.sak.integrasjoner.pdl.PersonopplysningerService
+import no.nav.familie.ba.sak.integrasjoner.pdl.internal.PersonInfo
+import no.nav.familie.ba.sak.kjerne.behandling.domene.Behandling
+import no.nav.familie.ba.sak.kjerne.grunnlag.personopplysninger.Person
+import no.nav.familie.ba.sak.kjerne.grunnlag.personopplysninger.PersonopplysningGrunnlagRepository
+import no.nav.familie.kontrakter.felles.personopplysning.FORELDERBARNRELASJONROLLE
+import no.nav.familie.kontrakter.felles.personopplysning.Ident
+import org.springframework.stereotype.Service
+
+
+@Service
+class FiltreringsreglerService(
+        private val personopplysningerService: PersonopplysningerService,
+        private val personopplysningGrunnlagRepository: PersonopplysningGrunnlagRepository,
+        ) {
+
+    fun hentDataOgKjørFiltreringsregler(
+            morsIdent: String,
+            barnasIdenter: Set<String>,
+            behandling: Behandling,
+    ): FiltreringsreglerResultat {
+
+        val personopplysningGrunnlag = personopplysningGrunnlagRepository.findByBehandlingAndAktiv(behandling.id)
+                                       ?: throw IllegalStateException("Fant ikke personopplysninggrunnlag for behandling ${behandling.id}")
+
+        val mor = personopplysningGrunnlag.søker
+        val barnaFraHendelse = personopplysningGrunnlag.barna.filter { barnasIdenter.contains(it.personIdent.ident) }
+        val restenAvBarna = finnRestenAvBarnasPersonInfo(morsIdent, barnaFraHendelse)
+        val morLever: Boolean = !personopplysningerService.hentDødsfall(Ident(morsIdent)).erDød
+        val barnLever: Boolean = !barnasIdenter.any { personopplysningerService.hentDødsfall(Ident(it)).erDød }
+        val morHarIkkeVerge: Boolean = !personopplysningerService.harVerge(morsIdent).harVerge
+
+        return evaluerFiltreringsregler(
+                mor,
+                barnaFraHendelse,
+                restenAvBarna,
+                morLever,
+                barnLever,
+                morHarIkkeVerge
+        )
```

Comment:
Lurer på om det er like oversiktlig om alt bare settes direkte
```suggestion
        return evaluerFiltreringsregler(
                mor = personopplysningGrunnlag.søker
                barnaFraHendelse = personopplysningGrunnlag.barna.filter { barnasIdenter.contains(it.personIdent.ident)}
                restenAvBarna = finnRestenAvBarnasPersonInfo(morsIdent, barnaFraHendelse)
                morLever = !personopplysningerService.hentDødsfall(Ident(morsIdent)).erDød
                barnLever = !barnasIdenter.any { personopplysningerService.hentDødsfall(Ident(it)).erDød }
                morHarIkkeVerge  = !personopplysningerService.harVerge(morsIdent).harVerge
        )
```